### DISCUSSION TITLE
Latest build for Android

### DISCUSSION THREAD

**Discussion by mat9v** [Alex313031/Thorium-Special#64] [General]
This build in 64bit version will not start on Motorola Edge 30 Ultra on Android 13. 32bit version runs well (if slow, I mean Speedometer 2.0 reports 75 points while standard Chrome based on 114 gets 110 points)

**Comment by gz83** [comment]
What are the indications that the 64-bit version does not work? Is it to exit immediately after startup?

At the same time, we will also try to make the browser's performance better.

@mat9v

**Reply by mat9v** [reply]
It starts and immediately closes, other versions (32 bit one and shell in both 32 and 64 flavors) work without problems.
